sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/job_service.py

```diff
@@ -48,18 +52,24 @@ def __init__(
         self.job_service_type = job_service_type
         self.nodes = nodes
         self.status = status
+        self.public_keys = public_keys
         self.port = port
         self.properties = properties
         self._validate_nodes()
 
     def _to_rest_object(self) -> RestJobService:
+        properties = self.properties
```

Comment:
If we add it to JobService. We may want to restrict this property to only SSH job_service_type and throw validation error for any other cases.